plugins/fileservices/fileservices.cpp

```diff
@@ -273,53 +273,56 @@ static const char *getAccessibleEspServerURL(const char *param, IConstWorkUnit *
     {
         Owned<IConstEnvironment> daliEnv = openDaliEnvironment();
         Owned<IPropertyTree> env = getEnvironmentTree(daliEnv);
-        StringBuffer wsfsurl;
-        if (env.get()) {
-            Owned<IPropertyTreeIterator> iter1 = env->getElements("Software/EspProcess");
-            ForEach(*iter1)
+
+        if (env.get())
+        {
+            StringBuffer wsFSUrl;
+            StringBuffer espInstanceComputerName;
+            StringBuffer bindingProtocol;
+            StringBuffer instanceAddressXPath;
+            StringBuffer instanceAddress;
+
+            Owned<IPropertyTreeIterator> espProcessIter = env->getElements("Software/EspProcess");
+            ForEach(*espProcessIter)
             {
-                Owned<IPropertyTreeIterator> iter2 = iter1->query().getElements("EspBinding");
-                ForEach(*iter2)
+                Owned<IPropertyTreeIterator> espBindingIter = espProcessIter->query().getElements("EspBinding");
+                ForEach(*espBindingIter)
                 {
-                    Owned<IPropertyTreeIterator> iter3 = iter2->query().getElements("AuthenticateFeature");
-                    ForEach(*iter3)
+                    espBindingIter->query().getProp("@service",wsFSUrl.clear());
+                    if (wsFSUrl.length() && (stricmp(wsFSUrl.str(),"EclWatch")==0 || stricmp(wsFSUrl.str(),"ws_fs")==0))
```

Comment:
Not a big deal, but I find strieq() easier to read.

Also this is why occasionally a more generic string name is good... if its being used for multiple purposes.  This name is no longer self describing.